landoapi/models/landing_job.py

```diff
@@ -28,7 +29,10 @@ class LandingJobStatus(enum.Enum):
     column of `LandingJob`.
     """
 
-    # Initial creation state.
+    # Initial state.
+    CREATED = "CREATED"
+
+    # Ready state.
```

Comment:
NOTE: this may cause problems when migrating, since Alembic does not gracefully support changes to existing enums.

TODO: Evaluate if there is an alternate/better way of doing this either here on in the migration file itself.